awx/api/serializers.py

```diff
@@ -5460,14 +5460,24 @@ def get_percent_capacity_remaining(self, obj):
     def get_health_check_pending(self, obj):
         return obj.health_check_pending
 
-    def validate(self, data):
-        if self.instance:
-            if self.instance.node_type == Instance.Types.HOP:
```

Comment:
are we losing this validation for some reason? It seems this could still apply for VM installation as HOP nodes don't have the requisite packages to do execution.

Reply:
Thank you for raising that point! Should it be renamed since we have another method called `validate`?